src/utils/validation.ts

```diff
@@ -79,6 +79,18 @@ export const validateLimitedAmount = (amount: string, decimals?: number, max?: s
   }
 }
 
+export const validateDecimalLength = (value: string, maxLen?: number, minLen = 1) => {
+  if (!maxLen || !value.includes('.')) {
+    return
+  }
```

Comment:
This is not validating large numbers without a `.` e.g. 1000000000000000000000000000

Reply:
Whole numbers don't have decimals.